src/test/kotlin/sirius/biz/storage/s3/ObjectStoresTest.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.storage.s3
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import sirius.kernel.SiriusExtension
+import sirius.kernel.commons.Files
+import sirius.kernel.commons.Tuple
+import sirius.kernel.di.std.Part
+import java.io.File
+import java.io.FileOutputStream
+import java.net.URL
+import java.nio.file.Files as files_;
+import java.nio.charset.StandardCharsets
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+
+@ExtendWith(SiriusExtension::class)
+class ObjectStoresTest {
+
+    @Test
+    fun `createBucketWorks`() {
```

Comment:
Maybe use the capabilities of Kotlin here with method names that can include spaces so the test case description is a bit more readable.